src/test/java/com/afornalik/ox/board/TestBoardField.java

```diff
@@ -13,19 +13,21 @@
 
 
     public void shouldSetUpFieldValueToX() {
-        //then
-        BoardField boardField = new BoardField(FIELD_STATUS_O.X);
+        //given
+        BoardField boardField = new BoardField(FIELD_STATUS_X);
 
         //then
         assertEquals(boardField.showStatus(),FIELD_STATUS_X);
     }
```

Comment:
Wheres the "when"? And there are more examples. There must be a way to follow guidelines, for example make first line like "// given + when" or even declare field in given and initialize it in when

Reply:
Yes my mistake, anyway class was deleted along with BoardField.